gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -1123,7 +1112,7 @@ MM_VerboseHandlerOutput::handleGCOPOuterStanzaStart(MM_EnvironmentBase* env, con
 	}
 
 	char tagTemplate[200];
-	getTagTemplate(tagTemplate, sizeof(tagTemplate), manager->getIdAndIncrement(), type ,contextID, duration, omrtime_current_time_millis());
+	getTagTemplate(tagTemplate, sizeof(tagTemplate), manager->getIdAndIncrement(), type ,contextID, duration, omrtime_current_time_millis(), terminationReason);
```

Comment:
Once `terminationReason` is removed as param from this method, is it possible to remove from `getTagTemplate`? Is this the only call site to `getTagTemplate` using  `terminationReason`?